### ISSUE TITLE
Automatic create a new PR to Bump CLI version  

### ISSUE THREAD

**Issue by brustolin** [getsentry/sentry-fastlane-plugin#137]
After #136 we need a GitHub action that automatically creates a PR that bump the CLI version alongside the plugin version and runs CI. 
This way we have an easy way to pin CLI version with the plugin version and guarantee that the new CLI release didn't break the plugin.

**Comment by brustolin** [comment]
This is already done right @denrase ?

**Comment by denrase** [comment]
Yes!

**Comment by philipphofmann** [comment]
Was done with https://github.com/getsentry/sentry-fastlane-plugin/pull/144.
